src/main/java/org/jabref/logic/layout/TextBasedPreviewLayout.java

```diff
@@ -54,6 +55,11 @@ public String getText() {
         return text;
     }
 
+    @Override
+    public String getInternalName() {
+        return NAME;
+    }
+
     @Override
     public String getName() {
         return Localization.lang("Customized preview style");
```

Comment:
I would suggest you use getDisplayName() instead of getName() and getName() instead of getInternalName()